.github/workflows/cmake-configure.yml

```diff
@@ -0,0 +1,37 @@
+name: Configure (wasm32)
```

Comment:
Name of this file doesn't make sense... Maybe `configure-wasm32.yml` instead.

Reply:
Looks like I goofed when renaming. I'm going to combine the two workflows into one.

Reply:
I have cleaned it up and moved Emscripten into `cmake.yml` and `configure.yml`, it is much better than originally because I am using a GitHub Action to simplify installing Emscripten.